skiplang/compiler/src/skipExpand.sk

```diff
@@ -332,10 +332,11 @@ fun populateDefinitions(
         name = key match {
         | SKStore.SID(x) -> x
         };
+        defFilesArray = defFiles.collect(Array);
```

Comment:
Is `collect(Array)` the best choice here? What about `collect(Vector)`?

Reply:
It doesn't matter for the iteration, but `setArray` expects an array